qa/python_models/init_args/model.py

```diff
@@ -42,13 +39,10 @@ def check_init_args(args):
         'model_instance_device_id':
             '0',
         'model_repository':
-            '/opt/tritonserver/qa/L0_backend_python/models/init_args',
+            os.getenv("TRITON_DIR", "/opt/tironserver") + '/qa/L0_backend_python/models/init_args',
```

Comment:
Where is the env `TRITON_DIR` being set? Is it in the gitlab yaml file?

Reply:
The directory value is defined in our `test.sh` script:
https://github.com/triton-inference-server/server/blob/mchornyi-r23.02-jp/qa/L0_backend_python/test.sh#L42

Reply:
For testing on jetson, I was wondering where the `TRITON_DIR` is defined for the correct path on jetson runners? i.e. for jetson runners, the `TRITON_DIR` would be `/raid/home/gitlab-runner/builds/FeEZGYLm/0/dl/dgx/tritonserver/tritonserver`

Reply:
@krishung5  please ping me tomorrow we have to address this comment by referring to the internal repository.

Reply:
Synced offline with Misha. Resolving.